api/client/proto/authservice.proto

```diff
@@ -802,13 +802,33 @@ message DeleteMFADeviceWithTokenRequest {
     string DeviceID = 2;
 }
 
+// AddMFADeviceWithTokenRequest is a request to add a mfa device for the user defined in
+// token.
+message AddMFADeviceWithTokenRequest {
+    // TokenID is the ID of a user token.
+    string TokenID = 1;
+    // DeviceName is the name of the device.
+    string DeviceName = 2;
+    // SecondFactorCred is the new mfa device auth credentials.
+    oneof SecondFactorCred {
```

Comment:
Let's use MFARegisterResponse here, so it plays nicer with existing API and ongoing MFA changes.